tests/Schema/TableTest.php

```diff
@@ -1768,4 +1768,22 @@ public function testInvalidPrimaryKeyIndex(): void
         $this->expectException(InvalidState::class);
         $table->getPrimaryKeyConstraint();
     }
+
+    public function testOverwritingForeignKeyConstraint(): void
+    {
+        $table = Table::editor()
+            ->setUnquotedName('foo')
+            ->setColumns(
+                Column::editor()
+                    ->setUnquotedName('id')
+                    ->setTypeName(Types::INTEGER)
+                    ->create(),
+            )
+            ->create();
+
+        $table->addForeignKeyConstraint('bar', ['id'], ['id'], ['foo' => 'bar']);
```

Comment:
`['foo' => 'bar']` is not a valid set of options, and they shouldn't be necessary for the test. Please remove.